fastjet/internal/plot/delaunay_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2017 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package plot
+
+import (
+	"bytes"
+	"image"
+	"io/ioutil"
+	"os"
+	"reflect"
+	"testing"
+
+	"go-hep.org/x/hep/fastjet/internal/delaunay"
+)
+
+func TestDelaunayPlot(t *testing.T) {
+	fname := "delaunay_example.png"
+	ref := "testdata/delaunay_example.png"
```

Comment:
convention (from gonum/plot) is to name that `"testdata/delaunay_golden.png"`.

Reply:
done